### DISCUSSION TITLE
Github is taking author as kapilraj1234, although on my local system its only kapil

### DISCUSSION THREAD

**Discussion by Kapil987** [community/community#30905] [Other Feature Feedback, Questions, & Ideas]
# git config file
[user]
        name = kapil
        email = kapil0123@gmail.com

when code is pushed to git-hub the author for commit shows kapilraj1234, how to solve this problem

**Comment by spenserblack** [comment]
GitHub uses the committer *email* to pick which account is the committer. [It looks like some of your commits have a different email](https://api.github.com/repos/Kapil987/Python_prac/commits).
